src/net/ICE/RtpIceChannel.cs

```diff
@@ -292,7 +294,7 @@ internal int RTO
         {
             if (_dnsLookupClient == null)
             {
-                _dnsLookupClient = new DnsClient.LookupClient();
+                _dnsLookupClient = new DnsClient.LookupClient(DefaultNameServers?.ToArray());

```

Comment:
Pretty sure this will result in an `ArgumentNullException` if the `DefaultNameServers` aren't specified.

See:

https://github.com/MichaCo/DnsClient.NET/blob/c313562c001875c05c8d9a23b5408781005ac834/src/DnsClient/LookupClient.cs#L345

It'd be good to have a unit test in https://github.com/sipsorcery/sipsorcery/blob/master/test/unit/net/ICE/RtpIceChannelUnitTest.cs to catch such cases.

Reply:
This will indeed call the wrong constuctor